src/Curiosity/Runtime.hs

```diff
@@ -247,38 +247,21 @@ handleCommand runtime display command = do
       output <-
         runAppMSafe runtime . IS.execVisualisation $ Data.VisualiseFullStmDb
       display $ show output
-      return ExitSuccess
+      pure ExitSuccess
     Command.SelectUser select -> do
       output <- runAppMSafe runtime . IS.execVisualisation $ Data.VisualiseUser
         select
       display $ show output
-      return ExitSuccess
+      pure ExitSuccess
     Command.UpdateUser update -> do
       output <- runAppMSafe runtime . IS.execModification $ Data.ModifyUser
         update
       display $ show output
-      return ExitSuccess
+      pure ExitSuccess
     _ -> do
       display $ "Unhandled command " <> show command
       return $ ExitFailure 1
-
--- | Given an initial state, applies a list of commands, returning the list of
```

Comment:
This one is not redundant. I'd like to expose this functionality both in a command-line tool and on the web: given a script, runs it, but expose each intermediate state, possibly with the ability to diff between two consecutive state.